src/Balance.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * Balance.php
+ * php version 7.2.0
+ *
+ * @category Class
+ * @package  Xendit
+ * @author   Ellen <ellen@xendit.co>
+ * @license  https://opensource.org/licenses/MIT MIT License
+ * @link     https://api.xendit.co
+ */
+
+namespace Xendit;
+
+/**
+ * Class Balance
+ *
+ * @category Class
+ * @package  Xendit
+ * @author   Ellen <ellen@xendit.co>
+ * @license  https://opensource.org/licenses/MIT MIT License
+ * @link     https://api.xendit.co
+ */
+class Balance
+{
+    use ApiOperations\Request;
+
+    /**
+     * Send GET request to retrieve data
+     *
+     * @param string $account_type account type (CASH|HOLDING|TAX)
+     * @param array  $headers      user's headers
+     *
+     * @return array
+     * @throws Exceptions\ApiExceptions
+     */
+    public static function getBalance($account_type = null, $headers = [])
+    {
```

Comment:
Do PHP have enum? We can create account_type enum if it is possible. If it is not possible, how do we handle the error if the account_type is invalid?

Reply:
PHP doesn't have enumeration, so for the previous version it will return format error instead from Xendit's API error. In the latest commit, I added in the code so it will throw an InvalidArgumentException instead.